packages/features/schedules/lib/use-schedule/useSchedule.ts

```diff
@@ -36,17 +41,14 @@ export const useSchedule = ({
       // if `prefetchNextMonth` is true, two months are fetched at once.
       endTime: (prefetchNextMonth ? nextMonthDayjs : monthDayjs).endOf("month").toISOString(),
       timeZone: timezone!,
-      eventTypeId: eventId!,
       duration: duration ? `${duration}` : undefined,
+      eventTypeId: eventId,
+      org,
+      isTeamEvent,
     },
     {
       refetchOnWindowFocus: false,
-      enabled:
-        Boolean(username) &&
-        Boolean(eventSlug) &&
-        (Boolean(eventId) || eventId === 0) &&
-        Boolean(month) &&
-        Boolean(timezone),
+      enabled: Boolean(username) && Boolean(eventSlug) && Boolean(month) && Boolean(timezone),
```

Comment:
we don't depend on eventTypeId now so we don't wait for useEvent request to return response and then start with this query (slots.getSchedule) and get eventTypeId on slots.getSchedule